### ISSUE TITLE
Force closes and patching issues

### ISSUE THREAD

**Issue by Mysteryagr** [chenxiaolong/DualBootPatcher#683]
Hello
I built the app for many MT6580 devices
After I struggled to configure Travis CI
Check here: https://github.com/Mysteryagr/DualBootPatcher

And the users who are using my builds are having the following issues:
1- App FC when they select roms tap
2- Unable to grant root permission due to FC
3- patching rom returns error code 200
4- App FC at the end of patching the rom

Here is a part of the logcat:
```
12-31 19:19:53.319  4117  4176 V MbtoolConnection: minimum version: 9.1.0
12-31 19:19:53.369  4117  4176 E RomConfig: Failed to load /storage/emulated/0/MultiBoot/primary/config.json
12-31 19:19:53.369  4117  4176 E RomConfig: java.io.FileNotFoundException: /storage/emulated/0/MultiBoot/primary/config.json: open failed: ENOENT (No such file or directory)
12-31 19:19:53.369  4117  4176 E RomConfig:     at libcore.io.IoBridge.open(IoBridge.java:452)
12-31 19:19:53.369  4117  4176 E RomConfig:     at java.io.FileInputStream.<init>(FileInputStream.java:76)
12-31 19:19:53.369  4117  4176 E RomConfig:     at java.io.FileInputStream.<init>(FileInputStream.java:103)
12-31 19:19:53.369  4117  4176 E RomConfig:     at java.io.FileReader.<init>(FileReader.java:66)
12-31 19:19:53.369  4117  4176 E RomConfig:     at com.github.chenxiaolong.dualbootpatcher.RomConfig.loadFile(RomConfig.java:221)
12-31 19:19:53.369  4117  4176 E RomConfig:     at com.github.chenxiaolong.dualbootpatcher.RomConfig.getConfig(RomConfig.java:79)
12-31 19:19:53.369  4117  4176 E RomConfig:     at com.github.chenxiaolong.dualbootpatcher.RomUtils.loadConfig(RomUtils.java:300)
12-31 19:19:53.369  4117  4176 E RomConfig:     at com.github.chenxiaolong.dualbootpatcher.RomUtils.getRoms(RomUtils.java:275)
12-31 19:19:53.369  4117  4176 E RomConfig:     at com.github.chenxiaolong.dualbootpatcher.switcher.service.GetRomsStateTask.execute(GetRomsStateTask.java:71)
12-31 19:19:53.369  4117  4176 E RomConfig:     at com.github.chenxiaolong.dualbootpatcher.switcher.service.BaseServiceTask.run(BaseServiceTask.java:125)
12-31 19:19:53.369  4117  4176 E RomConfig:     at com.github.chenxiaolong.dualbootpatcher.ThreadPoolService$1.run(ThreadPoolService.java:181)
12-31 19:19:53.369  4117  4176 E RomConfig:     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
12-31 19:19:53.369  4117  4176 E RomConfig:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
12-31 19:19:53.369  4117  4176 E RomConfig:     at java.lang.Thread.run(Thread.java:818)
12-31 19:19:53.369  4117  4176 E RomConfig: Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
12-31 19:19:53.369  4117  4176 E RomConfig:     at libcore.io.Posix.open(Native Method)
12-31 19:19:53.369  4117  4176 E RomConfig:     at libcore.io.BlockGuardOs.open(BlockGuardOs.java:186)
12-31 19:19:53.369  4117  4176 E RomConfig:     at libcore.io.IoBridge.open(IoBridge.java:438)
12-31 19:19:53.369  4117  4176 E RomConfig:     ... 13 more
```
please fix
Thanks in advance

**Comment by zacharee** [comment]
Same issue here on the LG V20 H918.

**Comment by chenxiaolong** [comment]
Fixed by #738.

**Comment by Mysteryagr** [comment]
Thanks :D
